### PR TITLE
API - Add the ability to provide generic CDRs

### PR DESCRIPTION
**Pull request by amonzerol**

### CODE REVIEW

File: ziro-cdr-template.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source Phone Number,Source user,Destination Phone Number,Destination user,Call Duration (in seconds),Start time (in ISO8601)
+5001,bdesnoyers,5002,ckarawani,525,2024-10-29T05:50:12Z
+5001,bdesnoyers,,+15149408284,384,2024-10-29T05:50:12Z
+5002,ckarawani,,+15149408000,1084,2024-10-29T05:50:12Z
```
Comment (philippemin) [inline ziro-cdr-template.csv]: > Upload a copy of the CSV file template to our pbx-exporter repo under ziro-cdr-template.csv.

Did benoit want example values in the template or an empty template? I'm not sure from the success criteria

**Review by philippemin — COMMENTED** [review]
just a question

File: ziro-cdr-template.csv (hunk 1 above)
Comment (amonzerol) [reply ziro-cdr-template.csv]: He wants values, but not these ones. I'll update with a new file soon

**Review by philippemin — APPROVED** [review]
👍

**Pull request merged**